### DISCUSSION TITLE
.NET - Can you provide some examples or API docs on LangChain.Providers.Amazon.Bedrock

### DISCUSSION THREAD

**Discussion by pratip-bagchi** [tryAGI/LangChain#289] [Q&A] [answered]
I am experimenting with RAG + C# code using Amazon Bedrock calling Haiku...but could not able to figure out any API docs or example on how to use them. 

Somehow I realized I should start with below code, bit did not find any means to pass model arguments like top_k or top_p etc. Any help will be appreciated. 

```
var provider = new BedrockProvider(RegionEndpoint.USEast1);
var llm = new AnthropicClaude3ChatModel(provider, claudeModelId);
```

**Comment by curlyfro** [comment, marked as answer]
hi Pratip,
here's an example of how to pass settings.

https://github.com/tryAGI/LangChain/wiki/Getting-started-with-Amazon-Bedrock

**Reply by curlyfro** [reply]
there are also lots of examples in the Bedrock integration tests.
